tikv/raftstore/fsm_store.go

```diff
@@ -461,78 +375,75 @@ func (bs *raftBatchSystem) start(
 		computeHashWorker: newWorker("compute-hash", wg),
 		wg:                wg,
 	}
-	builder := &raftPollerBuilder{
+	bs.ctx = &GlobalContext{
 		cfg:                  cfg,
+		engine:               engines,
 		store:                meta,
-		engines:              engines,
+		storeMeta:            newStoreMeta(),
+		storeMetaLock:        new(sync.RWMutex),
+		snapMgr:              snapMgr,
 		router:               bs.router,
-		splitCheckScheduler:  workers.splitCheckWorker.scheduler,
-		regionScheduler:      workers.regionWorker.scheduler,
-		raftLogGCScheduler:   workers.raftLogGCWorker.scheduler,
-		compactScheduler:     workers.compactWorker.scheduler,
-		pdScheduler:          workers.pdWorker.scheduler,
-		computeHashScheduler: workers.computeHashWorker.scheduler,
 		trans:                trans,
+		pdScheduler:          bs.workers.pdWorker.scheduler,
```

Comment:
Should we remove the `workers` fields and save each worker in `GlobalContext`

Reply:
workers are not used after start, we don't need them in `GlobalContext`.